vite.config.ts

```diff
@@ -1,12 +1,25 @@
 import eslint from '@nabla/vite-plugin-eslint';
+import { sentryVitePlugin } from '@sentry/vite-plugin';
 import react from '@vitejs/plugin-react-swc';
 import { defineConfig } from 'vite';
 import { configDefaults, coverageConfigDefaults } from 'vitest/config';
 
 export default defineConfig(({ mode }) => ({
   base: '/',
   envPrefix: 'REACT_APP_',
-  plugins: [react(), mode !== 'test' && eslint()],
+  plugins: [
+    react(),
+    mode !== 'test' && eslint(),
+    // Put the Sentry vite plugin after all other plugins
+    sentryVitePlugin({
```

Comment:
Using the local default envs gives quite a few warnings (auth token and org missing). Should the plugin be disabled by default for local? 🤔

Reply:
Yes good point. Changed the plugin to be enabled only when `SENTRY_AUTH_TOKEN` is provided.